### DISCUSSION TITLE
Support manual override for git blame

### DISCUSSION THREAD

**Discussion by devinrhode2** [gitkraken/vscode-gitlens#2647] [Ideas]
Rebasing is very difficult to learn, and laborious to do. I'll likely keep rebasing till the end of time because it's a better way to resolve merge conflicts.

At several jobs, I've run into debates about rebasing.

I've talked to a colleague about the topic for untold number of hours.

A key insight I've had, is that we should be able to explicit override git blame insights.

In some sense, code comments and git blame messages are the same thing, a very futuristic vcs/code management system would store that data in the same way, albeit "code comments" have a higher priority.

A simple way to store per-line git-blame comments would be git refs.

TL;DR: if there were some way to manually override traditional git blame messages, that would be fantastic.
